assets/blocks/diagonalLeft.block

```diff
@@ -0,0 +1,6 @@
+{
+    "displayName": "Wall Diagonal Left",
```

Comment:
Why did you create these blocks? I don't think that they are really necessary as they are just "proxy" blocks for `StructuralResources:TimberFrameDiagonalLeft`